### DISCUSSION TITLE
Allow more objects within expandable objects

### DISCUSSION THREAD

**Discussion by ShutdownRepo** [GitbookIO/community#73] [Feature Requests]
### What's your feature/idea?

Would it be possible to allow objects like hints and tabs (and possibly others) in expandable objects?

### What's your desired outcome?

Indicated above

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [ ] Medium
- [X] Low

**Comment by Katerlad** [comment]
Hints Is important for me to be in Expandable sections.

**Reply by taszap** [reply]
This is a duplicate/linked to #16

**Comment by taszap** [comment]
Just as heads up I will close this issue in favour of #16 to avoid duplication :)
